README.md

```diff
@@ -17,16 +17,14 @@ If you want to know more you can head over to our website: https://sabnzbd.org.
 
 SABnzbd has a good deal of dependencies you'll need before you can get running. If you've previously run SABnzbd from one of the various Linux packages, then you likely already have all the needed dependencies. If not, here's what you're looking for:
 
-- `python` (SABnzbd version 2: only python 2.7.x. SABnzbd version 3: python 3.5.x and higher, often called python3)
-- `python-cheetah`
+- `python` (Python 3.5 and higher, often called python3)
 - `par2` (Multi-threaded par2 installation guide can be found [here](https://sabnzbd.org/wiki/installation/multicore-par2))
 - `unrar` (Make sure you get the "official" non-free version of unrar)
 - `sabyenc` (installation guide can be found [here](https://sabnzbd.org/sabyenc))
 
 Optional:
```

Comment:
The optional modules stuff should be removed, see my prior comment.

Reply:
Hmm but how would we list something like `python-dbus` in `requirements.txt`?

Reply:
Something like this?
```
# optional, for system power management support
dbus-python; sys_platform != 'win32' and sys_platform != 'darwin'
```
Should be enough for anyone running from source as well as packagers to base their decisions on. If too many issues were to arise with dbus-python, we could simply comment it out by default later.

Reply:
That doesn't seem to work:
https://travis-ci.org/github/sabnzbd/sabnzbd/jobs/679656500#L219
Will mention that that it requires `dbus` to be installed, that seems to be the problem right?

The PyPi index also shows we could use the GObjects for it: https://pypi.org/project/dbus-python/

Reply:
The actual problem is this one:
> configure: error: Package requirements (dbus-1 >= 1.8) were not met:
>  No package 'dbus-1' found

Forgot the python3-dbus is an extension rather than pure-python. Probably needs `libdbus-1-dev` to build.